applications/sim_showers_for_trigger_rates.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/python3
+
+'''
+    Summary
+    -------
+    This application simulates showers to be used in trigger rate calculations.
+    Arrays with one (1MST) or four telescopes (4LST) can be used, in case of \
+    mono or stereo trigger configurations, respectively. 
+
+    Simulations are managed by the shower_simulator module.
+    Each run is simulated in a job. Each job is submitted by using the submission \
+    command from the global config settings (see config_template.yml). \
+    The config entry extraCommands can be used to extra commands to be ran in each job,
+    before the actual simulation.
+
+    At the moment, the shower simulations are performed by CORSIKA, which requires \
+    the zstd package. Please, make sure that the command to set your zstd path is \
+    properly set by the extraCommands in config.yml.
+
+    Command line arguments
+    ----------------------
+    array (str, required)
+        Name of the array (1MST, 4LST ...).
+    site (str, required)
+        South or North.
+    primary (str, required)
+        Name of the primary particle (proton, helium ...).
+    nruns (int, optional)
+        Number of runs to be simulated (default=100).
+    nevents (int, optional)
+        Number of events simulated per run (default=100000).
+    zenith (float, optional)
+        Zenith angle in deg (default=20).
+    azimuth (float, optional)
+        Azimuth angle in deg (default=0).
+    output (str, optional)
+        Path of the directory to store the output simulations. By default, \
+        the standard output directory will be used.
+    test (activation mode, optional)
+        If activated, no job will be submitted. Instead, an example of the \
+        run script willbe printed.
+    verbosity (str, optional)
+        Log level to print (default=INFO).
+
+    Example
+    -------
+    Producing a set of proton showers for trigger rate simulations of LST.
+
+    .. code-block:: console
+
+        python applications/sim_showers_for_trigger_rates.py -a 4LST -s North \
+        --primary proton --nruns 100 --nevents 10000 --output {some dir for large files}
```

Comment:
Maybe put in brackets what a large file is in this context.

Reply:
Is that needed here? You have right above the actual description of the arguments. This lines is just a simple example. Ideally all the arguments in the example would be set, but in this case that is not possible.

Reply:
Whenever you write 'large file' there is no information given. Is in 1 GB? 1 TB? What is large? Really not important, but comments should be 'maximal informational'.